Bug Tracking/Bug 1

```diff
@@ -0,0 +1,15 @@
+Title:Cart Empties on Logout

```

Comment:
Steps should be in a numbered list (1,2,3,etc).
Include an expected result for each step.